app/views/posts/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Posts List</h1>
+
+<p>Here is a list of all posts for a given user bas ed on the given user_id</p>
```

Comment:
p>Here is the list of all posts for a given user based on the given user_id</p>

Reply:
@Singh212dab You mean based !!